Form/ChoiceList/PhpcrOdmQueryBuilderLoader.php

```diff
@@ -93,8 +94,13 @@ public function getEntitiesByIds($identifier, array $values)
         $qb = clone $this->queryBuilder;
         $alias = $qb->getPrimaryAlias();
         $where = $qb->andWhere()->orX();
+
         foreach ($values as $val) {
-            $where->same($val, $alias);
+            if (UUIDHelper::isUUID($val)) {
+                $where->eq()->field($alias.'.uuid')->literal($val)->end();
```

Comment:
we don't have the model instance at this point yet, right? could we find the class from the query builder if its not explicitly specified?

btw, is this assuming that the uuid is mapped on the property `$uuid`? this is an assumption we should not make. maybe we just need an option to specify the uuid property to use with this form type, and if it is not set, we assume path?